src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -846,7 +846,10 @@ typedef enum {
     RTC_CODEC_VP8 = 3,                                                            //!< VP8 video codec.
     RTC_CODEC_MULAW = 4,                                                          //!< MULAW audio codec
     RTC_CODEC_ALAW = 5,                                                           //!< ALAW audio codec
-    RTC_CODEC_UNKNOWN = 6,
+    RTC_CODEC_H265 = 6,                                                           //!< H265 video codec
```

Comment:
Should we keep UNKNOWN as 6 and just add the rest afterwards?

Reply:
This is from the H265 contribution so I kept it the same. Your argument makes sense, I'll modify it

Reply:
Fixed!